examples/front_ends/simple_calculator_custom_routes/README.md

```diff
@@ -129,18 +166,61 @@ The server starts with both standard and custom endpoints:
 Access comprehensive request metadata:
 
 ```bash
-curl -X 'POST' \
-  'http://localhost:8000/get_request_metadata' \
+curl -X POST http://localhost:8000/get_request_metadata \
   -H 'accept: application/json' \
   -H 'Content-Type: application/json' \
   -H 'Authorization: Bearer token123' \
   -d '{"unused": "show me request details"}'
 ```
 
-Expected Response:
+Expected Response Format:
 
 <!-- path-check-skip-begin -->
 ```console
-{"value":"Method: POST, URL Path: /get_request_metadata, URL Scheme: http, Headers: {'host': 'localhost:8000', 'user-agent': 'curl/8.7.1', 'accept': 'application/json', 'content-type': 'application/json', 'authorization': 'Bearer token123', 'content-length': '37'}, Query Params: {}, Path Params: {}, Client Host: ::1, Client Port: 56922, Cookies: {}, Conversation Id: None"}
+{"value":"Method: POST, URL Path: /get_request_metadata, URL Scheme: http, Headers: {'host': 'localhost:8000', 'user-agent': 'curl/8.7.1', 'accept': 'application/json', 'content-type': 'application/json', 'authorization': 'Bearer token123', 'content-length': '95'}, Query Params: {}, Path Params: {}, Client Host: ::1, Client Port: 56922, Cookies: {}, Conversation Id: None, Request Body: {'message': 'show me request details', 'user_id': 123, 'tags': ['test', 'demo'], 'active': True}"}
```

Comment:
It seems odd that a request sent with name "unused" would be transformed to name of "message"

Is this correct/expected?